test/onnx/pytorch_test_common.py

```diff
@@ -107,13 +107,15 @@ def wrapper(self, *args, **kwargs):
     return skip_dec
 
 
-def skipScriptTest(min_opset_version=float("inf")):
+def skipScriptTest(min_opset_version=float("inf"), reason=""):
     def skip_dec(func):
         @functools.wraps(func)
         def wrapper(self, *args, **kwargs):
             self.is_script_test_enabled = self.opset_version >= min_opset_version
             if not self.is_script_test_enabled and self.is_script:
-                raise unittest.SkipTest("Skip verify test for TorchScript")
+                raise unittest.SkipTest(
+                    ": ".join(["Skip verify test for TorchScript", reason])
```

Comment:
Use an fstring?

Reply:
didn't like the extra logic of checking empty str otherwise introducing ": " for nothing, any cleaner solution?

Reply:
I see what is going on here. 🤔 ` "Skip verify test for TorchScript" + (f": {reason}" if reason else "")` but not exactly elegant

Reply:
Or: Default to "no reason provided" to encourage ppl to add a reason?

Reply:
I'm thinking breaking it by "." So we won't complicate ourselves :)